src/cloud-cli/meltano/cloud/README.md

```diff
@@ -0,0 +1,85 @@
+# Meltano Cloud CLI
+
+`meltano-cloud` is a standalone CLI tool for interacting with the Meltano Cloud API.
+
+## Configuration
+
+`meltano-cloud` can be configured via
+`meltano-cloud config set <setting name> <setting value>`.
+
+### Configuration File
+
+Meltano Cloud configuration settings and credentials are stored in a JSON file.
+`meltano-cloud` searches for this configuration file using [platformdirs](https://github.com/platformdirs/platformdirs).
+
+If a configuration file is found but the configuration it contains
+is invalid, `meltano-cloud` will raise an error rather than continuing its search.
+
+### Settings
+
+The following settings are configurable:
+
+#### `auth_callback_port`
+
+As part of authentication, `meltano-cloud` runs a lightweight, short-lived HTTP
+server on `localhost` to handle authentication callbacks. By default, this
+server runs on port 9999 but can be configured to another port using this setting.
+
+- Command line flag: `--auth-callback-port`
+- Environment variable: `MELTANO_CLOUD_AUTH_CALLBACK_PORT`
+
+#### `base_url`
+
+This is the base URL used for API calls to Meltano Cloud, `https://api.meltano.cloud`
+by default.
+
+- Command line flag: `--base-url`
+- Environment variable: `MELTANO_CLOUD_BASE_URL`
+
+#### `base_auth_url`
+
+This is the base URL used for authentication calls to Meltano Cloud,
+`https://auth.meltano.cloud` by default.
+
+- Command line flag: `--base-auth-url`
+- Environment variable: `MELTANO_CLOUD_BASE_AUTH_URL`
+
+#### `organization_id`
+
+This is the default ID for the Meltano Cloud organization to use in interacting with
+the Meltano Cloud API.
+
+- Command line flag: `--organization-id`
+- Environment variable: `MELTANO_CLOUD_ORGANIZATION_ID`
+
+#### `project_id`
+
+For project-specific commands, this is the project ID to use by default in interacting
+with the Meltano Cloud API.
+
+- Command line flag: `--project-id`
+- Environment variable: `MELTANO_CLOUD_PROJECT_ID`
```

Comment:
This is potentially confusing due to the name collision of Meltano Core `project_id` and the (internal) "Cloud Project ID". I prefer not to have any user-facing references to Cloud Project ID unless their clearly delineated as something different.

Can we remove from user-facing docs or maybe give a new name to this concept?

The obvious suggestion of `cloud_project_id` does result in the unfortunate "`MELTANO_CLOUD_CLOUD_PROJECT_ID`", but if we can work around this, I'd be fine with just calling it `cloud_project_id` - since that is different enough in my mind to still be clear from a variety of contexts.


FWIW, we don't have the ID listed here yet, but we should add it if we are going to make it user-facing: https://github.com/meltano/cloud-docs/blob/main/docs/concepts.md